SimpleCalculator-master/SimpleCalculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,32 @@
+using Moq;
+using FluentAssertions;
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Abstraction;
+using SimpleCalculator.Business.Enums;
+using SimpleCalculator.Business.OperatorBusiness.Operators;
+
+namespace SimpleCalculator.Tests;
+
+public class CalculatorTests
+{
+    [Theory]
+    [MemberData(nameof(CalculationData))]
+    public void Calculate_SumOperation_ReturnSum(int firstNum, int secondNum, int expected, OperatorEnum @enum,
```

Comment:
@enum
چیه داستانش؟

Reply:
It means you can use reserved words as variable names
say you want a class named class, since class is a reserved word, you can instead call your class class:
```cs
List<Student> @class = new List<Student>();
```